confgenerator/fluentbit/processors.go

```diff
@@ -38,6 +38,26 @@ func TranslationComponents(tag, src, dest string, translations []struct{ SrcVal,
 	return c
 }
 
+// TranslationMoveComponents translates SrcVal on key src to DestVal on key dest, if the dest key does not exist.
+// The original key is removed when translated.
+func TranslationMoveComponents(tag, src, dest string, translations []struct{ SrcVal, DestVal string }) []Component {
```

Comment:
`TranslationMoveComponents` and `TranslationComponents` looks similar enough that it might make more sense to combine them.

Reply:
Yeah, they are very similar. I combined them (which also meant modifying everywhere where TranslationComponents was used).